docs/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
I think this compiled file has to be removed

Reply:
No, it's the homepage of the docs.

Reply:
Or it will be. But it looks like generated output, which should not be in `docs/` but in `docs/build`? I'm a bit confused.

Reply:
yes, thanks, done.